modules/ROOT/partials/configuration/mergetags_list.adoc

```diff
@@ -0,0 +1,74 @@
+[[mergetags_list]]
+== `+mergetags_list+`
+
+`+mergetags_list+` is an object array that specifies the menu content used for merge tags insertion. Every object specifies the configuration of a submenu or a menu item.
+
+NOTE: If the `+mergetags_list+` option is not set, or contains no entries, both the merge tags toolbar button and the merge tags menu item are hidden. Merge tag autosuggestions are also disabled if the `+mergetags_list+` option is not set, or contains no entries.
+
+*Type:* `+Array+`
+
+*Default value:* `+undefined+`
```

Comment:
```suggestion
```
If something doesn't have a default then we don't specify it. This is also done because what is said here contradicts the type.